packages/kit/types.d.ts

```diff
@@ -104,10 +104,7 @@ export type GetSession<Context = any, Session = any> = {
 	({ context }: { context: Context }): Session | Promise<Session>;
 };
 
-export type Handle<Context = any> = ({
-	request,
-	render
-}: {
+export type Handle<Context = any> = (handle: {
```

Comment:
this is saying the argument to `Handle` is called `handle`?

Reply:
Yes, this also feels weird for me, I can't seem to find any mention about the actual object name in the docs too, only mentions "It receives the `request` object and `render` method, which calls SvelteKit's default renderer."

Maybe just a `input` or `incoming` like the others? 

https://github.com/sveltejs/kit/blob/11ec751b1158edf636b2ad28fd71706f43b62ec0/packages/kit/types.d.ts#L97-L101